test/integration/tests/4045-no-run-benchmarks/files/.gitignore

```diff
@@ -0,0 +1,3 @@
+.stack-work/
```

Comment:
Remove the .gitignore file from the repo, there is a global .gitignore which already ignores the needed paths.

Reply:
👍